notebooks/official/experiments/get_started_with_comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>newer version of the cell:</p><p> </p><p>import os</p><p> </p><p># The Vertex AI Workbench Notebook product has specific requirements</p><p>IS_WORKBENCH_NOTEBOOK = os.getenv("DL_ANACONDA_HOME")</p><p>IS_USER_MANAGED_WORKBENCH_NOTEBOOK = os.path.exists(</p><p>&nbsp;&nbsp;"/opt/deeplearning/metadata/env_version"</p><p>)</p><p> </p><p># Vertex AI Notebook requires dependencies to be installed with '--user'</p><p>USER_FLAG = ""</p><p>if IS_WORKBENCH_NOTEBOOK:</p><p>&nbsp;&nbsp;USER_FLAG = "--user"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done. </p>